src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -56,18 +52,16 @@ def name(self):
 
     def setup(self, builder: Builder):
         self.start_time = get_time_stamp(builder.configuration.time.start)
-        self.location = builder.data.load(data_keys.POPULATION.LOCATION)
         self.randomness = builder.randomness.get_stream(self.name)
+        self.household_migration = builder.components.get_component("household_migration")
         self.employer_address_id_count = 0
         self.columns_created = [
             "employer_id",
-            "employer_name",
-            "employer_address_id",
             "personal_income_propensity",
             "employer_income_propensity",
         ]
         self.columns_used = [
-            "address_id",
+            "tracked",
```

Comment:
In all the places where this is used would it not have queried to tracked simulants only there anyways?  I thought population view's default was tracked simulants?

Reply:
This column is added because we want to be able to record untracked simulants in the business details pipeline.